cypress/integration/lock-exceptions/user-batch-deletes-lock-exceptions.js

```diff
@@ -5,9 +5,7 @@ When(`the user clicks the 'Batch deletion' button`, () => {
 })
 
 Then('the user is taken to the batch deletion page', () => {
-    cy.findByRole('heading', {
-        name: 'Lock Exception | Batch Deletion',
-    }).should('exist')
+    cy.url().should('contain', '/lock-exceptions/batch-deletion')
```

Comment:
The 'batch deletion' UI was extracted from the lock exceptions page and converted into its own separate page with its own URL.